packages/boot/src/__tests__/acceptance/controller.booter.acceptance.ts

```diff
@@ -46,10 +46,6 @@ describe('controller booter acceptance tests', () => {
   }
 
   async function stopApp() {
-    try {
-      await app.stop();
-    } catch (err) {
-      console.log(`Stopping the app threw an error: ${err}`);
-    }
+    await app?.stop();
```

Comment:
Should we check the state first, for consistency with other `stopApp` implementations?

```suggestion
    if (app?.state === 'started') await app?.stop();
```

Reply:
In this case, app.start() is called.